src/creationView/CreateView.js

```diff
@@ -0,0 +1,2316 @@
+import { Localizer, ActionHelper } from "../common/ActionSdkHelper";
+
+let request;
+let questionCount = 0;
+let questions = new Array();
+let settingText = "";
+let questionSection = "";
+let opt = "";
+let lastSession = "";
+
+let optionKey = "";
+let addMoreOptionsKey = "";
+let choicesKey = "";
+let questionTitleKey = "";
+let checkMeKey = "";
+let nextKey = "";
+let backKey = "";
+let requiredKey = "";
+let dueByKey = "";
+let resultVisibleToKey = "";
+let resultEveryoneKey = "";
+let resultMeKey = "";
+let correctAnswerKey = "";
+let everyoneKey = "";
+let onlyMeKey = "";
+let showCorrectAnswerKey = "";
+let answerCannotChangeKey = "";
+let invalidDateTimeKey = "";
+let sureDeleteThisKey = "";
+let okKey = "";
+let cancelKey = "";
+let forQuizAtleastOneQuestionKey = "";
+let maxTenOptionKey = "";
+let atLeastTwoOptionsRequiredKey = "";
+let questionLeftBlankKey = "";
+let selectCorrectChoiceKey = "";
+let addQuestionKey = "";
+let uploadCoverImageKey = "";
+let attachmentSet = [];
+let coverImageKey = "";
+let clearKey = "";
+let invalidFileFormatKey = "";
+
+/* ******************************** Events ************************************** */
+/**
+ * @event Keydown and Click for add Question Section
+ */
+$(document).on({
+    keydown: function (e) {
+        let key = e.which;
+        if (key === 13 || key === 32) {
```

Comment:
Please don't use Hard Coding like 13/32, better to create a constants/enum.

Also please take care at other places as well and if needed please create a separate constant file for all constants